resident/resident-service/src/main/java/io/mosip/resident/handler/service/SyncAndUploadService.java

```diff
@@ -177,17 +173,6 @@ public String getFilename() {
 
 			}
 
-		} catch (FileNotFoundException e) {
-			logger.error(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.REGISTRATIONID.toString(),
-					registartionId,
-					ExceptionUtils.getStackTrace(e));
-			throw new BaseCheckedException(ResidentErrorCode.BASE_EXCEPTION.getErrorCode(), ResidentErrorCode.BASE_EXCEPTION.getErrorMessage(), e);
-
-		} catch (IOException e) {
-			logger.error(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.REGISTRATIONID.toString(),
-					registartionId,
-					ResidentErrorCode.IO_EXCEPTION.getErrorMessage() + ExceptionUtils.getStackTrace(e));
-			throw new BaseCheckedException(ResidentErrorCode.API_RESOURCE_ACCESS_EXCEPTION.getErrorCode(), ResidentErrorCode.API_RESOURCE_ACCESS_EXCEPTION.getErrorMessage(), e);
```

Comment:
Why these exceptions are removed ?

Reply:
They are for IO and File Not found exception. They are thrown by tokengenerator. Once i remove the token generator, they are not thrown and i removed it.